protocol/serialization-types/tests/test-serialization.rs

```diff
@@ -602,22 +629,36 @@ where
 macro_rules! block_path_test {
     ($typ:ty, $path:expr) => {
         for block in TEST_BLOCKS.values() {
+            let start = std::time::Instant::now();
             test_in_block::<$typ>(&block.value, &[$path]);
+            println!(
+                "block {} duration {:?}",
+                block.block_name,
+                std::time::Instant::now() - start,
+            );
         }
     };
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
can we add a comment describing why we need this dummy struct for later reference?

Reply:
yeah i'll add it